packages/components/src/lib/controllers/app-controller.js

```diff
@@ -493,7 +496,8 @@ export default class AppController {
         providers: [], // A list of resource providers
         queryStillActive: false, // it is for Persian case, when we canReset
 
-        mouseMoveOverrideUpdate: 1
+        mouseMoveOverrideUpdate: 1,
+        prioritizeDefaultLanguage: false
```

Comment:
This is a Vuex.Store variable that controlls when default language should ghave the highest priority